README

```diff
@@ -271,34 +267,49 @@ REQUIREMENTS:
         - strace             (used by test/test-functions)
         - capsh              (optional, used by test-execute)
 
+POLICY FOR SUPPORT OF DISTRIBUTIONS AND ARCHITECTURES:
+        systemd main branch and latest major or stable releases are generally
+        expected to compile on current versions of popular distributions (at
+        least all non-EOL versions of Fedora, Debian unstable/testing/stable,
```

Comment:
How about the latest CentOS stream with EPEL? This is purely a question. Not a request to add it.

Reply:
We backport latest systemd to centos stream (currently 8 and 9) so it'd definitely be nice to explicitly mention those here as well

Reply:
Added now.